group/views.py

```diff
@@ -22,25 +22,23 @@
 SOFTWARE.'''
 from django.http import HttpResponse
 from django.shortcuts import render_to_response, render
+from group.models import Group
+import datetime
+
+def get_running_contest(request, group_id):
+        
+        group = Group.objects.get(id = group_id)
+        all_contest = group.trace_contest.all()
+        all_running_contest_list = []
+        now = datetime.datetime.now()
+
+        for contest in all_contest:
+            start_time = contest.end_time.replace(tzinfo=None)
+            end_time = contest.start_time.replace(tzinfo=None)
+            if start_time > now:
+                if end_time < now:
```

Comment:
I prefer using "and" statement.
BTW, a running contest should be "start time earlier than(<) now and end time later than(>) now", your logic seems to be reversed.
(I suggest to write some comments so that the code is easier to read.)